### PR TITLE
3 | Llist cards

### LINKED ISSUE #3: list-cards
can't be implemented without - https://github.com/BUI-studies/buoy-landing/issues/1

Image

Image

### PR DESCRIPTION
**Pull request by ohiienko-r**
PR resolves https://github.com/BUI-studies/buoy-landing/issues/3

### CODE REVIEW

File: src/sections/StudyFormat/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import { FC } from 'react'
+import { studyFormatPropsList } from './helpers'
+import ytIco from '../../assets/yt-ico.png'
+import smallArrow from '../../assets/small-arrow.png'
+import './styles.scss'
+
+const StudyFormat: FC = () => {
+	return (
+		<section className="study-format">
```
Comment (yakovenkoandyUA) [inline src/sections/StudyFormat/index.tsx]: name class this section just 'courses', again in naming section 2 word

File: src/sections/StudyFormat/index.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+			<div className="study-format__container">
```
Comment (yakovenkoandyUA) [inline src/sections/StudyFormat/index.tsx]: why I see this div here? section up there have block properties

File: src/sections/StudyFormat/styles.scss [hunk 3]
```diff
@@ -0,0 +1,196 @@
+@import '../../scss/helpers/variable.scss';
+@import '../../scss/helpers/mixins.scss';
+
+.study-format {
+	color: #fff;
+	&__container {
```
Comment (yakovenkoandyUA) [inline src/sections/StudyFormat/styles.scss]: it is possible to make one common container class and not copy it from file to file

File: src/sections/StudyResults/index.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react'
+import { studyResults } from './helpers'
+import './styles.scss'
+
+const StudyResults: FC = () => {
+	return (
+		<section className="study-results">
+			<div className="study-results__container">
```
Comment (yakovenkoandyUA) [inline src/sections/StudyResults/index.tsx]: div for div

File: src/sections/StudyResults/styles.scss [hunk 5]
```diff
@@ -0,0 +1,109 @@
+@import '../../scss/helpers/variable.scss';
+@import '../../scss/helpers/mixins.scss';
+
+.study-results {
+	color: #fff;
+	&__container {
```
Comment (yakovenkoandyUA) [inline src/sections/StudyResults/styles.scss]: it is possible to make one common container class and not copy it from file to file

**Review by yakovenkoandyUA — CHANGES_REQUESTED** [review]

File: src/sections/StudyFormat/index.tsx (hunk 2 above)
Comment (ohiienko-r) [reply src/sections/StudyFormat/index.tsx]: It's just a limiting container

File: src/sections/StudyResults/index.tsx (hunk 4 above)
Comment (ohiienko-r) [reply src/sections/StudyResults/index.tsx]: Limiting container again

File: src/sections/StudyResults/styles.scss (hunk 5 above)
Comment (ohiienko-r) [reply src/sections/StudyResults/styles.scss]: Sure

File: src/sections/StudyFormat/index.tsx (hunk 2 above)
Comment (ohiienko-r) [reply src/sections/StudyFormat/index.tsx]: Removed this one

File: src/sections/StudyFormat/styles.scss (hunk 3 above)
Comment (ohiienko-r) [reply src/sections/StudyFormat/styles.scss]: created a common class

**Review by yakovenkoandyUA — APPROVED** [review]

**Pull request merged**